modules/rosa-adding-tags-cli.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags-cli{context}"]
+= Adding tags to a machine pool using the ROSA CLI
+
+You can add tags to a machine pool for your {product-title} cluster by using the ROSA command line interface (CLI).
```

Comment:
I would suggest to add some notes about the tags formats those cannot pass the validation
Key starts with `aws:` is preserved
Key cannot be longer than 128
Value cannot be longer than 256
And OCM preserved tags
```
					"red-hat-managed":     "true",
					"red-hat-clustertype": "osd",
					"Name":                "<name>",
					"kubernetes.io/cluster/<cluster infra id>: "owned",
				
```

Reply:
Does this note capture it? I wasn't sure if I needed to include the specific ones you mentioned or should I just cover it vaguely like I did:

![image](https://github.com/openshift/openshift-docs/assets/16167833/524aa683-70e5-43de-bf1c-92385b0eabed)

Reply:
@EricPonvelle Looks good. Do we have any place to list the `any other reserved Opeshift Cluster Manager tags`?

Reply:
It is probably best to say something like:

> You must ensure that your tag keys are not `aws`, `red-hat-managed`, `red-hat-clustertype`, or `Name`. In addition, you must not set a tag key that begins with `kubernetes.io/cluster/`. Your tag's key cannot be longer than 128 characters, while your tag's value cannot be longer than 256 characters. Red Hat reserves the right to add additional reserved tags in the future.

Reply:
@michaelryanmcneill Thanks for the suggestion; I have incorporated it into the docs

![image](https://github.com/openshift/openshift-docs/assets/16167833/3cecc02b-6821-45e5-8f8e-c2050dea5549)

@xueli181114 does that work?

Reply:
@EricPonvelle Looks good